profiles/full.props

```diff
@@ -1,33 +1,25 @@
-    <TestHelpersTargetFrameworks>net462;netstandard2.0;netstandard2.1;net6.0;net8.0</TestHelpersTargetFrameworks>
-    <ApprovalTestsTargetFrameworks>net8.0</ApprovalTestsTargetFrameworks>
-    <RecipesTargetFrameworks>net462;netcoreapp2.1;netcoreapp3.1;net6.0;net8.0</RecipesTargetFrameworks>
+    <TestTargetFrameworks>net462;netcoreapp2.1;netcoreapp3.1;net6.0;net8.0</TestTargetFrameworks>
   </PropertyGroup>
   <PropertyGroup Condition="$([MSBuild]::IsOSPlatform('Linux'))">
-    <FakeItEasyTargetFrameworks>netstandard2.0;netstandard2.1;net6.0;net8.0</FakeItEasyTargetFrameworks>
-    <ValueTaskExtensionsTargetFrameworks>netstandard2.0;netstandard2.1;net6.0;net8.0</ValueTaskExtensionsTargetFrameworks>
-    <SpecsTargetFrameworks>netcoreapp2.1;netcoreapp3.1;net6.0;net8.0</SpecsTargetFrameworks>
-    <IntegrationTestsTargetFrameworks>netcoreapp2.1;netcoreapp3.1;net6.0;net8.0</IntegrationTestsTargetFrameworks>
-    <UnitTestsTargetFrameworks>netcoreapp2.1;netcoreapp3.1;net6.0;net8.0</UnitTestsTargetFrameworks>
-    <TestHelpersTargetFrameworks>netstandard2.0;netstandard2.1;net6.0;net8.0</TestHelpersTargetFrameworks>
-    <ApprovalTestsTargetFrameworks>net8.0</ApprovalTestsTargetFrameworks>
-    <RecipesTargetFrameworks>netcoreapp2.1;netcoreapp3.1;net6.0;net8.0</RecipesTargetFrameworks>
+    <TestTargetFrameworks>netcoreapp2.1;netcoreapp3.1;net6.0;net8.0</TestTargetFrameworks>
   </PropertyGroup>
   <PropertyGroup Condition="$([MSBuild]::IsOSPlatform('OSX'))">
-    <FakeItEasyTargetFrameworks>netstandard2.0;netstandard2.1;net6.0;net8.0</FakeItEasyTargetFrameworks>
-    <ValueTaskExtensionsTargetFrameworks>netstandard2.0;netstandard2.1;net6.0;net8.0</ValueTaskExtensionsTargetFrameworks>
-    <SpecsTargetFrameworks>net6.0;net8.0</SpecsTargetFrameworks>
-    <IntegrationTestsTargetFrameworks>net6.0;net8.0</IntegrationTestsTargetFrameworks>
-    <UnitTestsTargetFrameworks>net6.0;net8.0</UnitTestsTargetFrameworks>
-    <TestHelpersTargetFrameworks>net6.0;net8.0</TestHelpersTargetFrameworks>
+    <TestTargetFrameworks>net6.0;net8.0</TestTargetFrameworks>
+  </PropertyGroup>
+  <PropertyGroup>
+    <FakeItEasyTargetFrameworks>$(LibraryTargetFrameworks)</FakeItEasyTargetFrameworks>
+    <ValueTaskExtensionsTargetFrameworks>$(LibraryTargetFrameworks)</ValueTaskExtensionsTargetFrameworks>
+    <TestHelpersTargetFrameworks>$(LibraryTargetFrameworks)</TestHelpersTargetFrameworks>
     <ApprovalTestsTargetFrameworks>net8.0</ApprovalTestsTargetFrameworks>
-    <RecipesTargetFrameworks>net6.0;net8.0</RecipesTargetFrameworks>
+    <SpecsTargetFrameworks>$(TestTargetFrameworks)</SpecsTargetFrameworks>
+    <IntegrationTestsTargetFrameworks>$(TestTargetFrameworks)</IntegrationTestsTargetFrameworks>
+    <UnitTestsTargetFrameworks>$(TestTargetFrameworks)</UnitTestsTargetFrameworks>
+    <RecipesTargetFrameworks>$(TestTargetFrameworks)</RecipesTargetFrameworks>
   </PropertyGroup>
-</Project>
+</Project>
```

Comment:
Very cool.
It occurs to me know that we could, in the csprojs, probably replace `FakeItEasyTargetFrameworks` with `LibraryTargetFrameworks`, etc., and further simplify the `.props` files. If we wanted. Until we needed to introduce more exceptions again.

Reply:
It might look like 

https://github.com/FakeItEasy/FakeItEasy/commit/00a04665d9e67771b33f8016fd2864157a020925

Weirdly, I get complaints from the packer when I build locally, but nothing when [the workflow runs](https://github.com/blairconrad/FakeItEasy/actions/runs/11272613138).

Reply:
> Until we needed to introduce more exceptions again.

It's the main reason I didn't do it. I think it's good to keep the flexibility, I just wanted to simplify things a bit.